services/poeditor/poeditor.service.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { nonNegativeInteger } = require('../validators')
+const { coveragePercentage } = require('../color-formatters')
+const { BaseJsonService } = require('..')
+
+const documentation = `
+  <p>
+    You must specify the read-only API token from the POEditor account to which the project belongs.
+  </p>
+  <p>
+    As per <a href="https://poeditor.com/docs/api">the POEditor API documentation</a>,
+    <q>all requests to the API must contain the parameter api_token. You can get this key from your POEditor account.
+    You'll find it in <a href="https://poeditor.com/account/api">My Account > API Access</a>.</q>
```

Comment:
I don't have a POEditor account so I don't know exactly what the wording should be, but can we make the instruction here _super clear_ about generating a read-only token.
e.g: "tick the 'read-only' box and click 'generate token' "(or whatever the step is)

Reply:
I mean there’s nothing to do, it’s literally just there on that page:

![image](https://user-images.githubusercontent.com/6126377/77703586-7c1c0500-6fbb-11ea-8540-7c7cfad464aa.png)

Reply:
I've pushed a tiny wording change just to mention read-only again. It is important our docs try to help ensure users don't expose a token that allows write access (it still happens).